uisdk/src/main/java/com/karhoo/uisdk/screen/booking/booking/payment/adyen/AdyenDropInService.kt

```diff
@@ -14,7 +14,7 @@ class AdyenDropInService : DropInService() {
     }
 
     private fun getPayment(paymentComponentData: JSONObject) {
-        val requestString = createPaymentRequestString(paymentComponentData.getJSONObject("paymentMethod"))
+        val requestString = createPaymentRequestString(paymentComponentData)
         KarhooApi.paymentsService.getAdyenPayments(requestString).execute { result ->
```

Comment:
Would be nice to move the logic outside this class to ensure SRP, could be a presenter/view and treat this class as the View.
I understand we are trying to make it work and it is easier to keep it here for the time being

Reply:
As above